src/curated-corpus/components/RejectItemModal/RejectItemModal.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { Prospect, ProspectType } from '../../../api/generatedTypes';
+import { ScheduledSurfaces } from '../../helpers/definitions';
+import { RejectItemModal } from './RejectItemModal';
+
+describe('The RejectItemModal component', () => {
+  const prospect: Prospect = {
+    id: 'test-id',
+    prospectId: 'test-prospect-id',
+    prospectType: ProspectType.Global,
+    scheduledSurfaceGuid: ScheduledSurfaces[0].guid,
+    url: 'www.test-prospect-url.com',
+    title: 'test-title',
+  };
+  const toggleModal = jest.fn();
+  const onSave = jest.fn();
+
+  it('should render successfully', () => {
+    render(
+      <RejectItemModal
+        prospect={prospect}
+        isOpen={true}
+        onSave={onSave}
+        toggleModal={toggleModal}
+      />
+    );
+
+    // fetch the modal's heading and assert it renders successfully
+    expect(
+      screen.getByText(/reject this item from inclusion in the curated corpus/i)
+    ).toBeInTheDocument();
+  });
```

Comment:
I wanted to added another assertion where I fetch the prospect's title and assert that the `RejectItemForm` component renders as well within this component but the react testing library is unable to fetch it.

The prospect's title is rendered in a `<h6>` dom element and the library is unable to fetch it for some reason. Someone else on SO also ran into this: https://stackoverflow.com/questions/72391039/getbyrole-not-able-to-fetch-h6-in-react-testing-library